code/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
```

Comment:
Alright, so we set up a `Sequelize` configuration that uses the `environment database` or the `config database` to initialize a new... session/object? essentially a context in which we can use those Sequelize methods.